### PR TITLE
[Documentation] Update and fix the documentation for 20.07 release.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR is trying to address issue #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,15 @@ National Cybersecurity R&D Laboratory (NCL)\
...
-Maintainer : Aris Cahyadi Risdianto - ariscr@ncl.sg
- 
-Supervisor : Prof. Ee-chien Chang - changec@comp.nus.edu.sg
+Project Leader: Aris Cahyadi Risdianto - ariscr@ncl.sg
+Project Investigator (PI) : Prof. Ee-Chien Chang - changec@comp.nus.edu.sg
  
 Contributors:
 - Aris Cahyadi Risdianto (NCL/NUS)
 - DeZhang Lee (NUS)
+- Lai Yong Rong ()
```
Comment (krusagiz) [inline README.md:33]: Could we remove the `()` for this contributor if we don't know his affiliation? Alternatively, we can put `(Independent)`.

File: Octo-Bot/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Octo-Bot Preparation and Developer Guide
+
+In order to understand how each independent bot is developed and built,
+please check each of the type of bot documentation below:
+
+1. [Generic web browsing bot using Python Scrapy](web-browsing-bot/README.md)
+2. [Javascript-enabled web browsing bot using Java Selenium](javascript-web-bot/README.md)
+3. [Customized web browsing bot for Healthcare environment using Python Selenium](healthcare-web-bot/README.md)
+4. [Customized registration bot for Healthcare environment using Python Selenium](healthcare-web-bot/README.md)
+5. [Packet generator bot for generating realistic traffic usign Python Scapy](packet-generator-bot/README.md)
```
Comment (joelczk) [inline Octo-Bot/README.md:10]: `using` is spelt wrongly

File: Docs/README.md [hunk 3]
```diff
@@ -54,15 +54,15 @@ Docker image is by pushing back the compiled code into the GitHub repository
...
 configuring specific build trigger.
 
-### Octo-App (Octobot Application System)
+### Octo-App (OCTOBOT Application System)
 
 Application preparation sub-component is required to install and configure the
 requirement as defined in the main repository such as package installation
 (Python Kubernetes API, UI bootstrap method ...), basic environment
-configuration (Python/BASH/NPM environment ...) and others. User/application
+configuration (Python/BASH/NPM environment ...) and others. Third-party user/application
```
Comment (joelczk) [inline Docs/README.md:62]: I think `user` will be easier to understand.

File: README.md [hunk 4]
```diff
@@ -24,12 +24,15 @@ National Cybersecurity R&D Laboratory (NCL)\
 National University of Singapore (NUS)\
 Singapore
 
-Maintainer : Aris Cahyadi Risdianto - ariscr@ncl.sg
- 
-Supervisor : Prof. Ee-chien Chang - changec@comp.nus.edu.sg
+Project Leader: Aris Cahyadi Risdianto - ariscr@ncl.sg
+Project Investigator (PI) : Prof. Ee-Chien Chang - changec@comp.nus.edu.sg
```
Comment (krusagiz) [inline README.md:28]: Should this be `Principal Investigator (PI)` instead of `Project Investigator (PI)`?

File: Docs/README.md (hunk 3 above)
Comment (ariscahyadi) [reply Docs/README.md:62]: We have several users. Third-party user is considered as "operator" or "application".

**Pull request merged**